packages/field-base/src/delegate-focus-mixin.js

```diff
@@ -81,6 +81,9 @@ const DelegateFocusMixinImplementation = (superclass) =>
     /** @protected */
     _focusElementChanged(element, oldElement) {
       if (element) {
+        if (this.disabled) {
+          element.disabled = true;
+        }
```

Comment:
Shouldn't it always update the `disabled` property on a lazily added element?

```suggestion
        element.disabled = this.disabled;
```

Reply:
Good point. I think we can indeed change this to override the `disabled` set on the input.

Reply:
Done.